### ISSUE TITLE
Please consider porting this to extension Chrome.

### ISSUE THREAD

**Issue by w8sdz** [rgh36167/FfxDarkenBackground#3]
I really enjoy using this Add-on with Firefox. I wish it was also available for Chrome.
73 de w8sdz - Keith

**Comment by rgh36167** [comment]
Right now I don't use Chrome on the desktop but will consider to convert the code to WebExtensions style which should make it  easily portable to Chrome.

**Comment by w8sdz** [comment]
Thank you for your reply. I noticed from the Javascript source code that it looks a lot like a Greasemonkey script. If so, perhaps you could release a Greasemonkey version which would also work in Tampermonkey on Chrome.

**Comment by w8sdz** [comment]
You could distribute it on Greasyfork.org .

**Comment by rgh36167** [comment]
It is very similar. It has been some time that I wrote greasemonkey scripts and I have no clue if it is possible to have user configurable parameters for them? Otherwise users would have to edit the script or live with the defaults.

**Comment by w8sdz** [comment]
Yes, it's possible to configure parameters. Here are two examples:
[https://greasyfork.org/scripts/404-mouseover-popup-image-viewer](url)
[https://greasyfork.org/scripts/413-youtube-link-title](url)

I hope that helps.

**Comment by rgh36167** [comment]
Seems like those scripts coded it for themselves and there are more solutions like that, eg http://www.last.fm/group/Greasemonkeys/forum/35794/_/435575

Probably not worth the effort as anyone using greasemonkey can edit the values in the script.
